spaces/S000139/README.md

```diff
@@ -4,16 +4,19 @@ name: Countable bouquet of circles
 aliases:
 - $\mathbb R$ with $\mathbb Z$ collapsed to a point
 - Countable wedge sum of circles
+- Rose with infinitely many petals
```

Comment:
That's undetermined.  How about "Rose with countably many petals"  (no need to say "countably infinitely many")

Reply:
Perhaps "Infinite rose with countably-many petals"? See also [Rose (topology) on Wikipedia](https://en.wikipedia.org/wiki/Rose_(topology)).

Reply:
Could be iffy, as each "rose" is an infinite set already.
But thinking about it some more, I now think that "Rose with infinitely many petals" is fine.  It is not a completely determined description, but that does not matter.  It's only an alias anyway.

Reply:
Another option: "Rose with $\omega$-many petals".